ofxdump/getopt.h

```diff
@@ -0,0 +1 @@
+#include "../lib/getopt.h"
```

Comment:
I don't think this file is needed or should be added - it rather confuses stuff. If the header is included from within the ofxdump directory, please use a relative include at source code level.